galaxykubeman/values.yaml

```diff
@@ -192,7 +192,7 @@ galaxy:
                     <div class="container">
                         <div class="row mb-4">
                             <div class="col-md-12">
-                                <h2>Welcome to Galaxy on AnVIL</h2>
+                                <h2>Welcome to Galaxy</h2>
```

Comment:
Meaghan and Mike confirmed this version of the welcome text is ok on all Terra sites